test/matchmaker/match.test.data.js

```diff
@@ -0,0 +1,146 @@
+const testMatches = [{
+    id: 0,
+    player1:
+    {
+        id: 'PLAYER10',
+        firstname: 'Gerhard Copy',
+        lastname: 'Acker',
+        clubname: 'ESV SF Neuaubing',
+        gamesWon: 3,
+        matchIds: [0],
+        opponentIds:[],
+        qttr: 1515,
+        active: true,
+        hasFreeTicket: false
+    },
+    player2:
+    {
+        id: 'PLAYER30',
+        firstname: 'Ulrich Copy',
+        lastname: 'Dietzel',
+        clubname: 'TTC Friedberg ',
+        gamesWon: 1,
+        matchIds: [0],
+        opponentIds:[],
+        qttr: 1211,
+        active: true,
+        hasFreeTicket: false
+    },
+    sets: [{
+        set1: {
+            p1Points: 11,
+            p2Points: 4
+        },
+        set2: {
+            p1Points: 15,
+            p2Points: 13
+        },
+        set3: {
+            p1Points: 11,
+            p2Points: 5
+        },
+    }],
+    freeTicket: false
+},
+{
+    id: 1,
+    player1:
+    {
+        id: 'PLAYER1',
+        firstname: 'Gerhard',
+        lastname: 'Acker',
+        clubname: 'ESV SF Neuaubing',
+        gamesWon: 3,
+        matchIds: [1],
+        opponentIds:[],
+        qttr: 1415,
+        active: true,
+        hasFreeTicket: false
+    },
+    player2:
+    {
+        id: 'PLAYER2',
+        firstname: 'Achim',
+        lastname: 'Amthor',
+        clubname: 'SC Baldham-Vaterstetten ',
+        gamesWon: 5,
+        matchIds: [1],
+        opponentIds:[],
+        qttr: 1251,
+        active: true,
+        hasFreeTicket: false
+    },
+    sets: [{
```

Comment:
Mein Vorschlag zum speichern der Ergebnisse.

Ich würde nur fertige setzte speichern, also ist ein Satz gespielt so pusht man den nurnoch in das .sets array.
Oder anders?

Reply:
Hm, das sollten wir uns noch mal anschauen, schau dir mal [diesen PR an](https://github.com/mrvary/ping-pong-ponyhof/pull/24). das betrifft auch den Kommentar bezüglich `gamesWon`, das brauchen wir nicht doppelt schreiben, das hat Marco schon geschrieben :) über die matches Sache denk ich noch mal nach, sollten wir noch mal diskutieren

Reply:
Da liegst du richtig. Wir sollten Mittwoch noch mal die Strukturen durchsprechen, danach könnten wir  tatsächlich die Verteilung der Matches angehen